ECFR-XML-User-Guide.md

```diff
@@ -36,7 +36,7 @@ The purpose of this document is to provide an overview of e-CFR XML files and as
 
 **Q. What is the data set available for the e-CFR in XML?**
 
-**A.        ** E�CFR files in XML have been converted and simplified from data in SGML that originates at the OFR.  The E�CFR is available in XML starting with the year 2015. Only the most recent e-CFR data will be available at endpoints on the FDsys Bulk Data Repository. Please see [www.ecfr.gov](http://www.ecfr.gov/) to search and browse a point in time edition of the e-CFR.
+**A.        ** E–CFR files in XML have been converted and simplified from data in SGML that originates at the OFR.  The E–CFR is available in XML starting with the year 2015. Only the most recent e-CFR data will be available at endpoints on the FDsys Bulk Data Repository. Please see [www.ecfr.gov](http://www.ecfr.gov/) to search and browse a point in time edition of the e-CFR.

```

Comment:
[www.ecfr.gov](http://www.ecfr.gov/) to search and browse a point in time edition of the e-CFR.